src/bits/core/catalog_init.py

```diff
@@ -10,7 +10,7 @@
 
 import databroker
 
-from ..utils.config_loaders import iconfig
+from bits.utils.config_loaders import iconfig
```

Comment:
Getting puzzled here. If this part is in bits (and not instrument), then keep the relative path.

Reply:
Comment applies below, too